yarn.lock

```diff
@@ -6418,6 +6418,11 @@
   resolved "https://registry.npmjs.org/@uniswap/v2-core/-/v2-core-1.0.1.tgz"
   integrity sha512-MtybtkUPSyysqLY2U210NBDeCHX+ltHt3oADGdjqoThZaFRDKwM6k1Nb3F0A3hk5hwuQvytFWhrWHOEq6nVJ8Q==
 
+"@uniswap/v2-core@^1.0.1":
```

Comment:
this should be dedup'ed. run `yarn deduplicate`.

Reply:
deduplicate did not create a diff to yarn.lock

Reply:
I needed to update @uniswap/v2-periphery to be able to remove